docs/guarantor.md

```diff
@@ -19,7 +19,8 @@ The guarantor obtains rewards from staking, which is influenced by the following
 - Stake Limit: The Stake Limit determines the upper limit of the effective stake of CRUs by the validator, which is directly related to the amount of storage provided by the validator. In Maxwell, a 1TB SRD file corresponds to an upper limit of 1CRU, and a meaningful file corresponds to 1-5 times the upper limit of SRD, with specific ‘times’ related to the number of duplicates of meaningful files. For details, please refer to [DSM Guidance](DSM.md);
 - Effective Stake: The Effective Stake is **the amount of staking that actually generates incomes**, which is related to the stake limit of the validator that is guaranteed. An example is provided as follows:
 
-> If a validator obtains a staking income of 1000 CRUs and has his own staking share accounting for 90% of the total stake (the other 10% is the guarantor’s share). If the validator sets the guarantee fee at 95%, then the guarantor will get a total of 95 CRUs (1000 * 10% * 95% =95), with the remaining 5 CRUs going to the validator.
+> For example, if the validator obtains 900 CRU staking rewards and 100 CRU authoring reward (block generation reward) in one era, and if his self-staking accounts for 90% of his total staking(The remaining 10% is contributed by guarantors). If the guarantee fee is 95%, then the guarantors finally obtains (900 + 100) * 10% * 95% = 95 CRUs income, and the remaining 905 CRUs belongs to the validator.
```

Comment:
The sequential of these two paragraphs is different in English version and Chinese version.